### PR TITLE
Update cgp-0063.md

### PR DESCRIPTION
**Pull request by product-matt**
added description for governance proposal

### CODE REVIEW

File: CGPs/cgp-0063.md [hunk 1]
```diff
@@ -1,51 +1,78 @@
 ---
 cgp: 63
-title: Increase gas price
-date-created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
-author: <a list of the author's or authors' name(s) and/or username(s), or name(s) and email(s), e.g. (use with the parentheses or triangular brackets): FirstName LastName (@GitHubUsername), FirstName LastName <foo@bar.com>, FirstName (@GitHubUsername) and GitHubUsername (@GitHubUsername)>
```
Comment (martinvol) [inline CGPs/cgp-0063.md:5]: Add your name :)

File: CGPs/cgp-0063.md [hunk 2]
```diff
@@ -1,51 +1,78 @@
...
 
-### Status
-- DRAFT = Feedback collection
-- PROPOSED = The governance proposal was submitted to the network
-- EXECUTED = The governance proposal was executed on the network
-- EXPIRED = The governance proposal expired
-- WITHDRAWN = The governance proposal is withdrawn before being submitted on the network
+### Current Situation
+Currently, gas pricing on Celo roughly works as follows: 
+* Every block a gas_price_minimum is defined (accessible through the GasPriceMinimum smart contract). If the block is full (measured in gas), gas_price_minimum increases, if the block is empty, gas_price_minimum decreases (until gas_price_minimum_(lower_bound) is reached). These increases and decreases can be exponential, and thus the value of gas_price_minimum can change quickly.
```
Comment (martinvol) [inline CGPs/cgp-0063.md:27]: Not really full but exceeds the density target

File: CGPs/cgp-0063.md [hunk 3]
```diff
@@ -1,51 +1,78 @@
...
+* In any given block, only transactions specifying a gas_price higher or equal than gas_price_minimum are valid
+* The gas paid by users is split between validators (tip) and the community fund (base)
```
Comment (martinvol) [inline CGPs/cgp-0063.md:29]: and when the price is at minimum, in reality the majority goes to the community fund.

Comment (product-matt) [reply CGPs/cgp-0063.md:29]: at minimum, I believe 100% goes to community fund? I've added equations for tip and base which makes this clear.

File: CGPs/cgp-0063.md (hunk 2 above)
Comment (product-matt) [reply CGPs/cgp-0063.md:27]: adding "denisty target" as word as well - was mostly to describe _roughly_ how it works..

Comment (product-matt) [reply CGPs/cgp-0063.md]: I've made the commit but cannot see the changes above, if there's a better way to respond to comments, let's discuss 

511e2e444713f333b2634deb85edb6e7e496f872

File: CGPs/cgp-0063.md [hunk 4]
```diff
@@ -1,51 +1,78 @@
...
 
-Fill out the following template for each transaction in the proposal
 
-1. Description of transaction 1
-  - Destination: A human readable description of the address and method being called
-  - Data: A human readable description of the transaction data
-  - Value: How much CELO is being sent, and why?
+1. Setting a new gas price minimum
+  - Destination: GasPriceMinimum.setGasPriceMinimumFloor
+  - Data: ["25,000,000,000"]
```
Comment (martinvol) [inline CGPs/cgp-0063.md:66]:
```suggestion
  - Data: ["25000000000"]
```

**Review by calicokittencat — APPROVED** [review]

**Review by willkraft — APPROVED** [review]

**Pull request merged**